pkg/controller/jenkins/configuration/base/reconcile.go

```diff
@@ -173,24 +183,30 @@ func (r *ReconcileJenkinsBaseConfiguration) ensureResourcesRequiredForJenkinsPod
 	}
 	r.logger.V(log.VDebug).Info("ConfigurationAsCode Secret and ConfigMap added watched labels")
 
-	if err := r.createRBAC(metaObject); err != nil {
+	httpServiceName := resources.GetJenkinsHTTPServiceName(r.Configuration.Jenkins)
+	if err := r.createService(metaObject, httpServiceName, r.Configuration.Jenkins.Spec.Service); err != nil {
 		return err
 	}
-	r.logger.V(log.VDebug).Info("Service account, role and role binding are present")
+	r.logger.V(log.VDebug).Info("Jenkins HTTP Service is present")
```

Comment:
why these changes ?

Reply:
reordering, this is from another PR I was working on